### PR TITLE
insert tables and gffa_utils updated

### PR DESCRIPTION
**Pull request by ShaiviGanatra**
insert_tables.py
- sentient_being, vehicle class, vehicle type and vehicle added with foreign keys dependencies.
- vehicle_type_id in vehicle says 1 for temp

gffa_utils.py
- Function get_index_from_url added to support search for foreign keys across tables while insertions using regex.

### CODE REVIEW

File: gffa_utils.py [hunk 1]
```diff
@@ -125,4 +127,26 @@ def convert_to_none(value):
...
+    tables while inserting foreign keys.
+
+    Parameters:
+        value (obj): url string to extract index from
+
+    Returns:
+        int: index extracted from the url
+    """
+    try:
+        print("TESTING")
```
Comment (arwhyte) [inline gffa_utils.py:145]: comment out `print()` function calls

File: gffa_utils.py [hunk 2]
```diff
@@ -125,4 +127,26 @@ def convert_to_none(value):
...
+        print(str(value))
+        if type(value) == str:
```
Comment (arwhyte) [inline gffa_utils.py:147]: Use `isinstance()` instead.

File: insert_tables.py [hunk 3]
```diff
@@ -79,6 +80,22 @@ def prepare_sentient_being_type_list(sentient_being_type):
...
+# --------------- PREPARE SENTIENT_BEING LIST ---------------
+
+def prepare_sentient_being_list(sentient_being):
+    sentient_being_list = []
+    sentient_being_list.append(gffa_utils.get_index_from_url(sentient_being["species"]))
+    sentient_being_list.append(gffa_utils.get_index_from_url(sentient_being["homeworld"]))
+    sentient_being_list.append(sentient_being["name"])
+    sentient_being_list.append(sentient_being["name"])
+    sentient_being_list.append("description")
+    sentient_being_attr = dict((k, sentient_being[k]) for k in ['height', 'mass', 'hair_color', 'skin_color', 'eye_color', 'birth_year', 'gender', 'films', 'species', 'vehicles', 'starships'])
```
Comment (arwhyte) [inline insert_tables.py:92]: convert this line to a dictionary comprehension rather than calling `dict()`.

File: insert_tables.py [hunk 4]
```diff
@@ -90,6 +107,22 @@ def extract_vehicle_class_list(vehicle_data):
...
 
+# --------------- PREPARE VEHICLE LIST ---------------
+
+def prepare_vehicle_list(vehicle):
+    vehicle_list = []
+    vehicle_list.append(1)
+    vehicle_list.append(vehicle["vehicle_class"])
+    vehicle_list.append(vehicle["model"])
+    vehicle_list.append("description")
+    vehicle_attr = dict((k, vehicle[k]) for k in ['model'])
```
Comment (arwhyte) [inline insert_tables.py:118]: Convert to dict comprehension.

**Review by arwhyte — CHANGES_REQUESTED** [review]

File: gffa_utils.py [hunk 5]
```diff
@@ -1,4 +1,6 @@
+import re
 import json
+from sys import intern
```
Comment (arwhyte) [inline gffa_utils.py:3]: Check this import and if unnecessary remove it.

**Review by arwhyte — CHANGES_REQUESTED** [review]

**Comment by ShaiviGanatra** [comment]
Updates as per datamodel 1.5

- unnecessary print and imports removed
- dict comprehension used for all tables
- isinstance() added in utils
- vehicle, vehicle_class and vehicle_type revised
- vehicle_class.vehicle_type_id pending

**Review by arwhyte — APPROVED** [review]

**Pull request merged**